src/Microsoft.DotNet.Cli.Utils/FileAccessRetryer.cs

```diff
@@ -47,5 +48,35 @@ public static class FileAccessRetrier
                 }
             }
         }
+
+        /// <summary>
+        /// Run Directory.Move and File.Move in Windows has a chance to get IOException with
+        /// HResult 0x80070005 due to Indexer. But this error is transient.
+        /// </summary>
+        internal static void RetryOnMoveAccessFailure(Action action)
+        {
+            const int ERROR_HRESULT_ACCESS_DENIED = unchecked((int)0x80070005);
+            int nextWaitTime = 10;
+            int remainRetry = 10;
+
+            while (true)
+            {
+                try
+                {
+                    action();
+                    break;
+                }
+                catch (IOException e) when (e.HResult == ERROR_HRESULT_ACCESS_DENIED)
+                {
+                    Thread.Sleep(nextWaitTime);
+                    nextWaitTime *= 2;
```

Comment:
this factor looks good?

Reply:
LGTM. That gives ~10s to complete 10 retries.

Reply:
👍